control-plane-operator/controllers/hostedcontrolplane/configoperator/reconcile.go

```diff
@@ -317,6 +326,9 @@ func buildHCCContainerMain(image, hcpName, openShiftVersion, kubeVersion string,
 		if platformType == hyperv1.IBMCloudPlatform {
 			c.Command = append(c.Command, "--controllers=controller-manager-ca,resources,inplaceupgrader,drainer,hcpstatus")
 		}
+		if externalInfra {
+			c.Command = append(c.Command, fmt.Sprintf("--infra-kubeconfig=%s", path.Join(volumeMounts.Path(c.Name, hccVolumeInfraKubeconfig().Name), kas.KubeconfigKey)))
+		}
```

Comment:
this append could be added to the refactor function that isolates all the kubevirt logic.